pages/guide.py

```diff
@@ -30,7 +30,7 @@
 # TODO(jrobbins): phase out gae_users
 from google.appengine.api import users as gae_users
 from framework import users
-from google.appengine.ext import db
+from google.appengine.ext import ndb
```

Comment:
I think this line could be deleted.  It does not seem to be referenced in the file.

Reply:
Okay... I have removed it. Thank you